tailwind.config.js

```diff
@@ -1,15 +1,23 @@
+const colors = {
+  gray: {
+    850: '#18202F',
+    950: '#000a14',
+  },
+  skyBlue: '#009bd2',
+  limeGreen: '#28d972',
+};
+
 module.exports = {
   content: ['./src/**/*.{js,ts,jsx,tsx}', './content/**/*.{md,mdx}'],
   darkMode: 'class',
   theme: {
     extend: {
       colors: {
-        gray: {
-          850: '#18202F',
-          950: '#000a14',
+        ...colors,
+        'logo-bg': {
+          DEFAULT: colors.skyBlue,
+          dark: '#29c6ff',
```

Comment:
Didn't define this in our [colors](https://github.com/mirainc/developer-docs/pull/14/files#diff-ddc1067f080930bc48865f4660316a14b21bbc9f9ba85975ce8479342b983d2eR1-R9) as it's a one-off.